README.md

```diff
@@ -0,0 +1,155 @@
+# Capsule sandbox
+
+The Capsule sandbox is a containerized development environment for building [Capsule](https://github.com/capsule-rs/capsule) applications in Rust.
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Getting Started](#getting-started)
+  * [Vagrant and VirtualBox](#vagrant-and-virtualbox)
+  * [Linux Distributions](#linux-distributions)
+  * [Without Docker](#without-docker)
+  * [Kernel NIC Interface](#kernel-nic-interface)
+* [Packaging for Release](#packaging-for-release)
+* [Contributing](#contributing)
+* [Code of Conduct](#code-of-conduct)
+* [License](#license)
+
+## Introduction
+
+`Capsule` is built on Intel's [Data Plane Development Kit](https://www.dpdk.org/) release 18.11. While it is written in Rust, it needs to be able to call `DPDK`'s C functions installed separately on the host as shared dynamic libaries. To make developing and running Capsule applications easy, we created the sandbox container with all the necessary dependencies. You can pull down the container and start writing network functions in stable `Rust` right away.
+
+## Getting Started
+
+To run the [sandbox](https://hub.docker.com/repository/docker/getcapsule/sandbox), the docker host must run on a Linux distribution. `DPDK` requires either Linux or FreeBSD. We plan to add FreeBSD support in the future.
+
+### Vagrant and VirtualBox
+
+The quickest way to start the sandbox is to use the Vagrant virtual machine included in this repository. If you are developing on either MacOS or Windows, this is the only way to write a `Capsule` application.
```

Comment:
I think we should link to the "sans Docker vm" section somewhere here... maybe as an aside?

Reply:
seems out of place. cuz the next section is still talking about docker. I will change the section heading to `Vagrant and Docker` instead, to make it clear this section is only docker based.